src/site/tools/pub/publishing.markdown

```diff
@@ -7,29 +7,81 @@ title: "Publishing a Package"
 
 # {{ page.title }}
 
-[Pub](/tools/pub) isn't just for using other people's packages. It also allows you to share
-your packages with the world. Once you've written some useful code and you want
-everyone else to be able to use it, just run:
+[Pub](/tools/pub) isn't just for using other people's packages.
+It also allows you to share your packages with the world. If you have a useful
+project and you want others to be able to use it, use the `pub publish`
+command. For your first time around, you can perform a dry run like so:
+
+{% prettify sh %}
+$ pub publish --dry-run
+{% endprettify %}
+
+Pub will check to make sure that your package follows the
+[pubspec format](pubspec.html) and
+[package layout conventions](package-layout.html), and then upload your package
+to [pub.dartlang.org](https://pub.dartlang.org). Pub will also show you all of
+the files it intends to "publish". Here's an example of publishing a package
+named `transmogrify`:
+
+{% prettify none %}
+Publishing transmogrify 1.0.0 to https://pub.dartlang.org:
+    .gitignore
+    CHANGELOG.md
+    README.md
+    lib
+        transmogrify.dart
+        src
+            transmogrifier.dart
+            transmogrification.dart
+    pubspec.yaml
+    test
+        transmogrify_test.dart
+
+Package has 0 warnings.
+{% endprettify %}
+
+To publish your package for reals, remove the `--dry-run` argument:
 
 {% prettify sh %}
 $ pub publish
 {% endprettify %}
 
-Pub will check to make sure that your package follows the [pubspec
-format](pubspec.html) and [package layout conventions](package-layout.html),
-and then upload your package to [pub.dartlang.org](http://pub.dartlang.org).
-Then any Pub user will be able to download it or depend on it in their
-pubspecs. For example, if you just published version 1.0.0 of a package named
-`transmogrify`, then they can write:
+After your package has been successfully uploaded to
+[pub.dartlang.org](https://pub.dartlang.org/), any Pub user will be able to
```

Comment:
Our convention is not to capitalize "pub" as a proper noun. There are a couple places you do so here, and they should be downcased.